src/main/java/uk/gov/hmcts/reform/ucmc/controllers/CallbackController.java

```diff
@@ -48,7 +48,7 @@ public CallbackResponse callback(
         @NotNull @RequestBody CallbackRequest callback,
         @PathVariable("version") Optional<CallbackVersion> version
     ) {
-        log.info("Received callback from CCD, eventId: {}", callback.getEventId());
+        log.info("Received callback from CCD, eventId: {} and callback type: {}", callback.getEventId(), callbackType);
```

Comment:
good change, but not relevant to this PR

Reply:
Wass removed